edm4hep.yaml

```
@@ -780,6 +780,36 @@ datatypes:
     OneToOneRelations:
       - edm4hep::Track track           // the corresponding track
 
+
+  #===== Generator related data =====
+
+  #---------- GeneratorEventParameters
+  edm4hep::GeneratorEventParameters:
+    Description: "Generator event parameters"
+    Author: "EDM4hep authors"
+    Members:
+      - double event_scale              // event scale
+      - double alphaQED                 // alpha_QED
+      - double alphaQCD                 // alpha_QCD
+      - int signal_process_id           // id of signal process
+      - double sqrt_s  [GeV]            // sqrt(s)
+    VectorMembers:
+      - double crossSections [pb]       // list of cross sections
+      - double crossSectionErrors       // list of cross section errors
```

```suggestion
      - double crossSectionErrors [pb]      // list of cross section errors
```
I assume they are also given in `[pb]`, but in any case they should have a unit.